pkg/cmd/dev/BUILD.bazel

```diff
@@ -59,3 +60,11 @@ go_test(
         "@com_github_stretchr_testify//require",
     ],
 )
+
+disallowed_imports_test(
+    "dev",
+    [
+        "@com_github_cockroachdb_errors//:errors",
```

Comment:
It feels like for something like dev (and, potentially, roachtest, roachprod, etc), we'd almost really want an *allowlist*, rather than having to play whack-a-mole with the forbidden imports. Just putting it out there in case that's easy to implement.

Reply:
(just a drive by, feel free to ignore)

Reply:
This is actually an existing use case of `VerifyNoImports`, so we will probably end up implementing the allowlist functionality either way. I'll take it as a follow-up.